policy/release/tasks.rego

```diff
@@ -0,0 +1,54 @@
+# METADATA
+# custom:
+#   tasks_required:
+#     rule_data:
+#       required_task_refs:
+#       - add-sbom-and-push
+#       - clamav-scan
+#       - deprecated-image-check
+#       - get-clair-scan
+#       - sanity-inspect-image
+#       - sanity-label-check
+#       - sanity-optional-label-check
+#       - sast-go
+package policy.release.tasks
+
+import data.lib
+import future.keywords.in
+
+all_tasks_missing_errors = {e | task := rego.metadata.chain()[_].annotations.custom.tasks_required.rule_data.required_task_refs[_]; e := {
```

Comment:
~Should this go in the `tasks_test.rego` file since it's only used there?~
No, because this needs access to the package annotations. I guess we could use `document` annotations but that seems overkill just for this use case. Maybe we can add a comment here explaining why `all_tasks_missing_errors` exists in this file to help us remember in the future?

Reply:
FYI in policy/pipeline/required_tasks_test.rego we do have the duplication that is being avoided here, but it's in a different form. Consider refactoring that so we use the same approach in both places. (Not a blocker for this PR, just something to consider.)